Formula/i2p.rb

```diff
@@ -7,7 +7,7 @@ class I2p < Formula
 
   bottle :unneeded
 
-  depends_on "openjdk"
+  depends_on :java => "1.8"
```

Comment:
Presumably `openjdk@11` won't work here?

Either way, the `install` script below will need updating as it still points to `openjdk`.

Reply:
I looked at their website and they said only java 1.8 was supported. Not sure if the test passing previously was a false positive. I'll check with openjdk@11.